am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -144,7 +149,8 @@ public void revokeResourceAccess(@NotNull @Valid ExplicitAccessMetadata accessMe
      */
     @AuditLog("filtered access to resource '{{resource.id}}' defined as '{{resource.definition.serviceName}}|"
         + "{{resource.definition.resourceType}}|{{resource.definition.resourceName}}' for accessor '{{userId}}' "
-        + "in roles '{{userRoles}}': {{result.access.accessManagementType}} access with {{result.access.permissions}}")
+        + "in roles '{{userRoles}}': {{result.access.accessType}} access with {{result.access.permissions}} "
+        + "and relationships {{result.relationships}}")
```

Comment:
```suggestion
        + "due to relationships {{result.relationships}}")
```

Reply:
also what happens when ROLE_BASED accessType? There will be no relationship correct?

Reply:
should this also be added to the grantAccess method?

Reply:
due to does not make sense to me here, I feel my version reads better

Reply:
@geek-soft when rolebased access this log comes up as "and relationships[]" is that ok with you?

Reply:
I guess we have no better solution. Relationship is just optional.

In terms of whether it should be null or empty list I'd say list as generally it is a good idea to have list always initiated.